.github/workflows/codegen.yaml

```diff
@@ -4,6 +4,10 @@ on:
   # See https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule
   - cron: '18 0-2 * * *'
   workflow_dispatch:
+    inputs:
+      batch_index:
+        type: number
+        description: Index of the batch ([0-2])
```

Comment:
Can you also add a note in the description that you need to run the job three times (once with each input) to run all the clients?